tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/Lambda.cs

```diff
@@ -0,0 +1,104 @@
+// <copyright file="Lambda.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS
+{
+    /// <summary>
+    /// CallTarget integration for AWS Handler
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "xxx",
+        TypeName = "yyy",
+        MethodName = "zzz",
+        ReturnTypeName = "System.Object",
+        ParameterTypeNames = new[] { "System.Object", "System.Object" },
+        MinimumVersion = "1",
+        MaximumVersion = "5",
+        IntegrationName = "AWSLambda")]
```

Comment:
+1, `[InstrumentMethod]` is only used to make sure our integrations (statically) make it into the InstrumentationDefinitions Payload. Since you're adding it dynamically, the attribute is not needed

Reply:
True, this attribute is not required if the integration will be added dynamically.